lib/Init.php

```diff
@@ -18,6 +18,9 @@ public static function initTheme()
         // Set to true to load all assets from a CDN if there is one specified
         Asset::loadFromCdn(false);
 
+        // Set to true to use the new Gutenberg editor in combination with Flynt components
+        Defaults::useGutenberg(true);
```

Comment:
I wasn't sure if this should be handled in the Default class or API class. any thoughts? I also thought about having a Gutenberg class and use it to configure and define variables on how Gutenberg is configured in Flynt.